scaleapi/__init__.py

```diff
@@ -1029,3 +1021,130 @@ def add_studio_assignments(self, emails, projects):
         }
         assignments = self.api.post_request(endpoint, payload)
         return [StudioLabelerAssignment(assigned_projects, email, self) for (assigned_projects, email) in assignments.items()]
+
+    def remove_studio_assignments(self, emails, projects):
+        """Removes projects from users based on emails.
+        Args:
+            emails (str):
+                emails to unassign
+            projects (str):
+                projects to unassign
+        
+        Returns:
+            Dict[StudioLabelerAssignment]
+        """
+        endpoint = "studio/assignments/remove"
+        payload = {
+            "emails": emails,
+            "projects": projects,
+        }
+        assignments = self.api.post_request(endpoint, payload)
+        return [StudioLabelerAssignment(assigned_projects, email, self) for (assigned_projects, email) in assignments.items()]
```

Comment:
if we look here: 
https://docs.google.com/document/d/1q9v4u5uOpvjHZSxv9p49787eqC49DkKZz2QC8A_GjHc/edit
might be better to use the google doc version of the documentation.
The return type of the `remove assignments` endpoint is not a list, but a dictionary. 

That means we cannot use list-comprehension (the python List/Array trick [func(x) for x in y])
We'll have to return a dictionary for this method